bookmarklet/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@code-with-durham/bookmarklet",
-  "version": "0.1.0",
+  "version": "0.2.3",
```

Comment:
How did you pick this specific version?

Reply:
I went through several iterations of committing versions that didn't work. I think the version 0.2.3 evokes a sense of progression and constant improvement.

Reply:
The versioning of packages has semantic meaning as npm packages generally follow semver. It’s fine for now, but going forward I recommend doing a minor version bump for a change like this since it’s backwards compatible.